systems/plants/@RigidBodyManipulator/contactConstraints.m

```diff
@@ -57,7 +57,8 @@
 % For now, all coefficients of friction are 1
 mu = ones(nC,1);
 
-d = obj.surfaceTangents(normal);
+d = surfaceTangentsmex(normal);
```

Comment:
unfortunately, it might not be quite this simple.  

in your cmakefile, surfaceTangentsmex is only built if eigen is found.  if eigen is not found (e.g. in the minimal distro), there might be a way to get here even if the mex file does not exist?

also, we always try to maintain the ability to pass symbolic variables through (for TaylorVar gradients, or for polynomial analysis).  So normal may not be numeric.

It's possible that one can only get to this part of the contactConstraints if bullet is enabled and with numeric types.. but i'm not sure.